package.json

```diff
@@ -40,6 +40,7 @@
     "enhanced-resolve": "^3.0.0",
     "eslint": "^6.0.1",
     "eslint-config-babel": "^9.0.0",
+    "eslint-import-resolver-node": "^0.3.2",
```

Comment:
What is it needed for? `eslint-plugin-import` (which, from what I understood, is the package using it) already has it in `dependencies`.